src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -1126,7 +1136,10 @@ def __str__(self):
 
     @property
     def is_required(self):
-        return self.type == self.REQUIRED
+        """
```

Comment:
Suggest we drop the docstring (most people won't see it, and it's not clear about when/how this will be deprecated) - better we log a [deprecation warning](https://github.com/django-oscar/django-oscar/blob/4644ede05a54e287a2445107b8ef000a70a842e4/src/oscar/core/decorators.py) in here.